src/libaktualizr/bootloader/bootloader.h

```diff
@@ -27,7 +31,7 @@ class Bootloader {
  private:
   const BootloaderConfig config_;
 
-  INvStorage& storage_;
+  std::shared_ptr<INvStorage> storage_;
```

Comment:
What was the reason for this change? shared_ptr makes it much harder to understand the lifetime of an object.

Reply:
> What was the reason for this change?

1) make it consistent with the way the other components "owns" a pointer to the storage instance
2) it imposes an additional burden to the bootloader clients to take care about the storage instance lifetime